app/DynamicsAdapter/DynamicsAdapter.Web.Test/MockLoggerExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.Logging;
+using Moq;
+using System;
+
+namespace DynamicsAdapter.Web.Test
+{
+    public static class MockLoggerExtensions
```

Comment:
can we change to MockLoggerExtensionsTest ?

What is this testing, the enricher extensions?

Reply:
This is not testing. This is an extension for verify log. 
It is used by SearchRequestJobTest and SearchResultServiceTest. When exception throws, it is used to verify the error is logged.